weave/trace/weave_client.py

```diff
@@ -1325,6 +1391,36 @@ def fail_call(self, call: Call, exception: BaseException) -> None:
         """Fail a call with an exception. This is a convenience method for finish_call."""
         return self.finish_call(call, exception=exception)
 
+    @trace_sentry.global_trace_sentry.watch()
+    def call_descendants(
+        self, call: Call, depth: int | None = None, limit: int | None = None
+    ) -> list[Call]:
+        """Get all descendant calls of a given call."""
+        res = self.server.calls_descendants(
+            CallsDescendantsReq(
+                project_id=self._project_id(),
+                call_ids=[call.id],
+                depth=depth,
+                limit=limit,
+            )
+        )
+        return list(res)
```

Comment:
Why not be lazy?  I think it's better to let the user have control -- the iterator we return already has list-like access